protocol/testutil/constants/affiliates.go

```diff
@@ -0,0 +1,32 @@
+package constants
+
+import (
+	affiliates "github.com/dydxprotocol/v4-chain/protocol/x/affiliates/types"
+)
+
+var (
+	DefaultAffiliateTiers = affiliates.AffiliateTiers{
+		Tiers: []affiliates.AffiliateTiers_Tier{
+			{
+				ReqReferredVolume:   0,
+				ReqStakedWholeCoins: 0,
+				TakerFeeSharePpm:    50000,
+			},
+			{
+				ReqReferredVolume:   1000000,
```

Comment:
This is in quote quantums, so 1_000_000_000_000 for 1million USDC. To avoid confusion could you change `ReqReferredVolume` to `ReqReferredVolumeQuoteQuantums`